internal/value/value_test.go

```diff
@@ -0,0 +1,177 @@
+package value
+
+import (
+	"encoding/json"
+	"math"
+	"testing"
+
+	"github.com/hashicorp/go-cty/cty"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBool(t *testing.T) {
```

Comment:
Aren't these different 3 test cases? The first tests that given a null cty val, the SUT produces nil. The second, that given a true cty val, the SUT produces a true bool. The third, that given a false cty val, the SUT produces a false bool. Should these be in the same test func, or should we structure them some other way?
Same applies for the rest of the tests in the file.

Reply:
Yep, that's a good point. We'll make it more clear 👍🏻